outputs.tf

```diff
@@ -17,8 +17,3 @@ output "public_ip_address" {
   description = "The public IP address of the application gateway"
   value       = var.disable_agw ? null : module.application_gateway[0].public_ip_address
 }
-
-output "app_configuration_endpoint" {
```

Comment:
> Removed output app_configuration_endpoint as it's not used.

By us but this could be a breaking change for someone else.

Reply:
By us, I've added it yesterday, but turns out it was unnecessary.